boards/milkv/duo/milkv_duo_cv1800b_c906_1.yaml

```diff
@@ -0,0 +1,14 @@
+identifier: milkv_duo/cv1800b/c906_1
```

Comment:
This should probably have the `ram` specified here. Used for testcase filtering.